.prettierrc.json

```diff
@@ -1,4 +1,13 @@
 {
   "printWidth": 110,
-  "trailingComma": "all"
-}
\ No newline at end of file
+  "trailingComma": "all",
+  "overrides": [
+    {
+      "files": "*.md",
+      "options": {
+        "printWidth": 80,
```

Comment:
Do we even want this to be 80?

Reply:
I don't mind. The setting exists because this is prettier default and we wanted our code lines to be longer than that. In cosmwasm we do not set the value and get 80 by default.

Reply:
My mild preference would be to use the default value everywhere, but I don't really mind either way.

Reply:
I think 80 is very short for code. What I used in other projects is 100, which can then work for code and text.